api/config/v1alpha1/envoyproxy_types.go

```diff
@@ -161,6 +161,29 @@ const (
 	LogComponentRuntime LogComponent = "runtime"
 )
 
+// ProxyBootstrap defines Envoy Bootstrap configuration.
```

Comment:
do we want to handle to list of `merges ?

Reply:
```
        // Merge merges the provided bootstrap with the default one. The provided bootstrap can add or override a value
	// within a map, or add a new value to a list.
	// Please note that the provided bootstrap can't override a value within a list.
	BootstrapTypeMerge BootstrapType = "Merge"
```

Reply:
I'm not sure whether there're any use cases that need a list of `Merge`.

Reply:
cool, yah, most likely not needed